admin/config/feature-toggles.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+/**
+ * List of feature to toggle on or off
+ * NB features should be written as a positive
+ * @type {{groupCreate: boolean}}
+ */
+
+module.exports = {
```

Comment:
as discussed - maybe do this as JSON, and we can have files such as....
`feature-toggles.dev.json`
`feature-toggles.uat.json`
etc
Add new env var to config.js so the target file can be overriden at runtime for each environment.
maybe consider employing ramda for merging overrides?